mainwindow.h

```diff
@@ -0,0 +1,22 @@
+#ifndef MAINWINDOW_H
+#define MAINWINDOW_H
+
+#include <QMainWindow>
+#include <QDesktopWidget>
+
+QT_BEGIN_NAMESPACE
+namespace Ui { class MainWindow; }
+QT_END_NAMESPACE
+
+class MainWindow : public QMainWindow
+{
+    Q_OBJECT
+
+public:
+    MainWindow(QWidget *parent = nullptr);
+    ~MainWindow();
+
+private:
+    Ui::MainWindow *ui;
```

Comment:
a nie lepiej z tego zrobić unique_ptr ?

Reply:
Ten kod został wygenerowany automatycznie

Reply:
i nic nie zmienialiscie w tym mainwindow.h  i mainwindow.cpp ?

Nawet jeśli nie, to i tak warto te uwagi zaaplikować. To, że kod sie wygenerował nie znaczy, że nie możemy go zmieniać i poprawiać

Reply:
dotyczy to wszyskich uwag